homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -206,6 +172,32 @@ def __init__(
         else:
             self._masked_id = None
 
+        self._apply_event(event)
```

Comment:
This is also called from outside after instantiating. We don't want side effects in init method so we should move this.

Reply:
Right